src/WebJobs.Script/Description/DotNet/DotNetFunctionInvoker.cs

```diff
@@ -15,20 +15,19 @@
 using System.Threading.Tasks;
 using Microsoft.Azure.WebJobs.Host;
 using Microsoft.Azure.WebJobs.Script.Binding;
-using Microsoft.Azure.WebJobs.Script.Diagnostics;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.Scripting;
 
 namespace Microsoft.Azure.WebJobs.Script.Description
 {
+    [CLSCompliant(false)]
```

Comment:
ExecutionContext is not CLS-Compliant. I think it's because the whole assembly from Extensions isn't explicitly marked as CLS Compliant. I've filed an issue: [https://github.com/Azure/azure-webjobs-sdk-extensions/issues/147](https://github.com/Azure/azure-webjobs-sdk-extensions/issues/147) 

---
In reply to: [76697897](https://github.com/Azure/azure-webjobs-sdk-script/pull/604#discussion_r76697897) [](ancestors = 76697897)